Makefile

```diff
@@ -73,7 +73,7 @@ $(STANDARD_ES_GRAMMAR_OUT): $(ECMA262_SPEC_HTML)
 	$(PYTHON) -m js_parser.extract_es_grammar $(ECMA262_SPEC_HTML) > $@ || rm $@
 
 rust: $(RS_AST_OUT) $(RS_TABLES_OUT)
-	cargo build
+	cargo build --features full
```

Comment:
:smile:

Reply:
I thought the flag was `--all`?

Reply:
what do you mean?

Reply:
I mean: `cargo build --all`

Reply:
okay, fixed in https://github.com/mozilla-spidermonkey/jsparagus/pull/409